package.json

```diff
@@ -91,7 +90,8 @@
     "decimal.js": "^10.3.1",
     "ethers": "^5.7.2",
     "lodash.get": "^4.4.2",
-    "superstruct": "^0.15.4"
+    "superstruct": "^0.15.4",
+    "tslib": "^2.6.2"
```

Comment:
This was the functional change to this file. Essentially what is happening is that when this `tslib` is listed as a dev dependency, it doesn't get injected into the `node_modules` file to be used by the `cjs` variant of `sdk-v2`. TSDX handled this, but now we have to do it manually.